static/style.css

```diff
@@ -1,124 +1,122 @@
-html, body, #wrapper{
+html, body, {
     height: 100%;
     width: 100%;
     margin: 0;
 }
 
-#cityMap{
+#mapCanvas{
     height: 100%;
     width: 100%;
     margin: 0;
+    text-align: center;
 }
 
+#navbar{
+  text-align: center;
+  padding: 10px;
+}
 
-ul#logo{
-  position: absolute;
-  top:0px;
-  display: flex;
+a#icon{
+  text-align: center;
+  opacity: 0.7;
 }
 
-ul#logo li{
-  text-align: left;
-  display: block;
+
+a#icon:hover{
   opacity: 1;
 }
 
-ul#navbar{
-  position: absolute;
-  top:0px;
-  margin-left: 350px;
-  display: flex;
+#sidePanel{
+  text-align: left;
+  padding-left: 20px;
+  width: 100%;
 }
 
-
-ul#navbar li{
-  text-align: centre;
-  padding: 0 25px;
-  display: block;
+#about{
+  margin:10px;
+  background-color: #555a66;
   opacity: 0.7;
+  height: 200px;
+  overflow: auto
 }
 
-ul#navbar li:hover{
-  opacity: 1;
+#displayInfo{
+  font-family: 'Roboto', sans-serif;
+  padding: 10px;
+  font-size: 15px;
+  color: white;
 }
 
-
-#displayInfo{
-    position: absolute;
-    font-family: 'Roboto', sans-serif;
-    background-color: #555a66;
-    opacity: 0.7;
-    padding: 10px;
-    font-size: 15px;
-    top: 150px;
-    left: 20px;
-    color: white;
-    width: 300px;
-    height: 200px;
+#stats{
+  margin:20px 10px 10px 10px;
+  background-color: rgb(255,255,255,0.7);
+  height: 400px;
```

Comment:
Same as the about section. I imagine the fun facts at the different levels (state, county, etc) will contain different information, and, as a result, different lengths. Better to let the div adjust to the height organically.